packages/lib/parse-dates.ts

```diff
@@ -1,23 +1,22 @@
-import type { I18n } from "next-i18next";
 import { RRule } from "rrule";
 
 import type { Dayjs } from "@calcom/dayjs";
 import dayjs from "@calcom/dayjs";
 import { detectBrowserTimeFormat } from "@calcom/lib/timeFormat";
 import type { RecurringEvent } from "@calcom/types/Calendar";
 
-import { parseZone } from "./parseZone";
+import { parseZone } from "./parse-zone";
 
-const processDate = (date: string | null | Dayjs, i18n: I18n) => {
+const processDate = (date: string | null | Dayjs, language: string) => {
```

Comment:
Existing code move from apps/web to reusable function. Only changed i18n argument to make it easier to use. Also updated implementation in web.